### PR TITLE
flake: Custom derivations for coq-core and coq-serapi

### PR DESCRIPTION
**Pull request by huwaireb**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -90,8 +119,13 @@
...
     };
 
+    coq = {
+      url = "github:ejgallego/coq/8e0314415cc3cd528fc81bf11075d294179f880a";
+      flake = false;
+    };
+
     coq-serapi = {
-      url = "github:ejgallego/coq-serapi/v8.17";
+      url = "github:ejgallego/coq-serapi/9c8e91fdad0182675e0cad34bdeba54e63db86cd";
```
Comment (Alizter) [inline flake.nix:111]: Is there a way to get this in the lock file so we only have to do `nix flake update` when we need to bump coq and serapi?

Comment (Alizter) [reply flake.nix:111]: Same for coq

Comment (huwaireb) [reply flake.nix:111]: Yes, if its fine for these inputs to be at the last commit we could remove the rev

**Comment by Alizter** [comment]
Thanks!

**Pull request merged**